SQL/New_patches/2021-07-28_diagnosis_evolution.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE `diagnosis_evolution` (
+  `DxEvolutionID` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `Name` varchar(255) DEFAULT NULL,
+  `ProjectID` int(10) unsigned DEFAULT NULL,
+  `visitLabel` varchar(255) DEFAULT NULL,
+  `instrumentName` varchar(255) DEFAULT NULL,
+  `sourceField` varchar(255) DEFAULT NULL,
+  `orderNumber` int(10) unsigned DEFAULT NULL,
+  PRIMARY KEY (`DxEvolutionID`),
+  UNIQUE KEY `TrajectoryName` (`Name`),
+  CONSTRAINT `FK_DxEvolution_1` FOREIGN KEY (`ProjectID`) REFERENCES `Project` (`ProjectID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE `candidate_diagnosis_evolution` (
```

Comment:
```suggestion
CREATE TABLE `candidate_diagnosis_evolution_rel` (
```

maybe?

Reply:
I think `rel` would imply that the diagnosis value itself is stored elsewhere. This table links to the configured Dx tracjectory, but the Diagnosis is only present here. not sure if that makes sense.

Reply:
I made the changes you requested.

There will be one row in the table per candidate/ DxEvolutionID. The latest Dx is just the one with highest orderNumber (but all are stored in the table)